app/controllers/account/job-opening/campaign.js

```diff
@@ -33,6 +33,12 @@ export default Controller.extend(addEdit, del, {
     close () {
       const model = this.get('model');
       model.set('closed', true);
+      if (!model.completedOn) {
+        model.set('completedOn', moment().toDate());
```

Comment:
Moment use here is pointless as we aren't doing anything with the date.
```suggestion
        model.set('completedOn', new Date());
```